node/engine/chainservice/eth_chainservice.go

```diff
@@ -183,11 +183,8 @@ func (ecs *EthChainService) listenForErrors(errChan <-chan error) {
 			ecs.wg.Done()
 			return
 		case err := <-errChan:
-			fmt.Println(err) // Print to STDOUT in case we're using a noop logger
 			ecs.logger.Error("chain service error", "error", err)
-
-			// Manually panic in case we're using a logger that doesn't call exit(1)
-			panic(err)
+			panic(err) // Manually panic in case we're using a logger that doesn't call exit(1)
```

Comment:
With the switch to `slog` we know that `ecs.logger.Error` will not `panic`, so we can remove the comment.
```suggestion
			panic(err)
```